tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "module": "esnext",
     "target": "es5",
-    "lib": ["es2020", "dom"],
+    "lib": ["es2020", "dom", "dom.iterable"],
```

Comment:
https://stackoverflow.com/a/71384592

I need to iterate over FormData in a test